devel/py-decorator4/Makefile

```diff
@@ -0,0 +1,22 @@
+PORTNAME=	decorator
+PORTVERSION=	4.4.2
+CATEGORIES=	devel python
+MASTER_SITES=	CHEESESHOP
+PKGNAMEPREFIX=	${PYTHON_PKGNAMEPREFIX}
+PKGNAMESUFFIX=	4
+
+MAINTAINER=	ygy@FreeBSD.org
+COMMENT=	Decorators for Humans
+
+LICENSE=	MIT
+LICENSE_FILE=	${WRKSRC}/LICENSE.txt
+
+USES=		python:3.6+
```

Comment:
`3.6+` is not really needed, but since it's a copy so it's also fine to keep it.

Reply:
Noted, I thought the same as well so kept them identical.